ArjunaJTS/jts/classes/com/arjuna/ats/internal/jts/orbspecific/interposition/coordinator/ServerTransaction.java

```diff
@@ -234,7 +234,7 @@ public final int doPhase2Commit ()
 
 		org.omg.CosTransactions.Status s = get_status();
 
-		if (s != org.omg.CosTransactions.Status.StatusPrepared) {
+		if (s != org.omg.CosTransactions.Status.StatusPrepared && s != org.omg.CosTransactions.Status.StatusCommitted) {
```

Comment:
yeah, I missed that, bottom-up would catch it so I can remove that bit - thanks for the review!